### ISSUE TITLE
Research

### ISSUE THREAD

**Issue by erinmgriffith** [erinmgriffith/Active-projects#11]
- [x] review documentation for Oracle SDK
- [x] review documentation for AWS RDS API.
- [ ] take tutorial on SQLPlus
- [x] begin mooc on three.js graphics/get reaquainted with javascript graphics
- [x] review jsp and servlets- get quick jsp reference.

**Comment by erinmgriffith** [comment]
AWS RDS API does not have a sample project in the sdk. After briefly reviewing the documentation I decided that the easiest way to go about the initial db development would be to use the Oracle Sql developer interface and then export it to the RDS instance I created.

**Comment by erinmgriffith** [comment]
After further experimentation  to run prototype 1.2 as a java web app, I have decided that it may be useful to forego the AWS RDS instance and instead utilize the oracle interface for glassfish that is included with netbeans. I can later use another hosting instance of AWS for App deployment if needed, and just stick with a oracle db posted to my go daddy address if not. 

This approach makes careful DB ER planning even more important, but also allows me to use plain old java servlets and the java.sql library instead of playing around with AWS RDS API that in my opinion has insufficient documentation.

I have moved the SQLplus tutorial task to a different milestone, as this may not be required at all and if so, definitely at a later faze.
